app/controllers/users_controller.rb

```diff
@@ -18,6 +18,7 @@ def index
 
   def new
     @user = User.new
+    @user.organization_users.build
```

Comment:
What is this line doing for us exactly?

Reply:
It builds the associated `OrganizationUser` record. Without this, the nested form won't render. See https://github.com/plataformatec/simple_form/wiki/Nested-Models